yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
why is this lockfile in the root?

Reply:
I think I may have accidentally run yarn install in the root before `cd`ing into website. Seems like something other people might do, I'll add to the root gitignore just in case.